### PR TITLE
fix labels helptext

### LINKED ISSUE #4599: tested 6519: wording change dashboard
**URL:** https://meinberlin-dev.liqd.net/dashboard/modules/burgerinnenhaushalt-3-phasen-5/labels/
**user:** initiator
**expected behaviour:** 
**behaviour:** the last sentence is wrong as long as react list is not released. 
**important screensize:**
**device & browser:** 
**Comment/Question:** temporarily take out last sentence from translation and mark for a later release that we have to take it back in.

Screenshot?
Bildschirmfoto 2022-10-21 um 13 55 49

### PR DESCRIPTION
**Pull request by khamui**
fixes #4599

### CODE REVIEW

File: locale/en_GB/LC_MESSAGES/djangojs.po [hunk 1]
```diff
@@ -70,46 +70,34 @@ msgstr ""
 "will be deleted if it does not meet our discussion rules (netiquette)."
 
 #: adhocracy4/comments/static/comments/Comment.jsx:96
-#: adhocracy4/comments_async/static/comments_async/comment.jsx:296
```
Comment (goapunk) [inline locale/en_GB/LC_MESSAGES/djangojs.po:73]: Seems like alot of js strings are getting removed again, however, they still seem to be in the code. I still don't really get why `make po` is so unreliable

Comment (khamui) [reply locale/en_GB/LC_MESSAGES/djangojs.po:73]: oh, i didn't doublecheck them tbh. Do you know what i can do about it?

Comment (goapunk) [reply locale/en_GB/LC_MESSAGES/djangojs.po:73]: no, sadly I have no idea. Last time @phillimorland just ran make po again and pushed it

Comment (goapunk) [reply locale/en_GB/LC_MESSAGES/djangojs.po:73]: maybe removing those strings is also correct and I just don't get it :sweat_smile:

Comment (fuzzylogic2000) [reply locale/en_GB/LC_MESSAGES/djangojs.po:73]: Yeah, that is the problem why we always put the strings on top in the jsx-Code. And then sometimes they are still gone. So, if there weren't changes and a lot of strings are lost you just have to check if they are still in the code and try `make po` again or ask someone else to do it or try to put the strings to the top. It's pretty annoying.

File: locale/de_DE/LC_MESSAGES/djangojs.po [hunk 2]
```diff
@@ -1365,6 +1292,18 @@ msgstr ""
...
+
+#~ msgid "Submit"
+#~ msgstr "Einreichen"
+
+#~ msgid "Other"
+#~ msgstr "Sonstige"
+
+#~ msgid "New answer"
+#~ msgstr "Neue Antwort"
+
```
Comment (fuzzylogic2000) [inline locale/de_DE/LC_MESSAGES/djangojs.po:1306]: These aren't that many translations lost. And they all seem to be from a4. Did you check if we still have the strings there?

Comment (khamui) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1306]: so it seems that they are still in use, some not in JSX and some of them are indeed used in JSX.
I checked with latest a4 main and the a4 commit which is used in current mB main?

curiosity: how do you know that they are from a4 @fuzzylogic2000 ?

Comment (fuzzylogic2000) [reply locale/de_DE/LC_MESSAGES/djangojs.po:1306]: I noticed that a lot of the lines that were removed were the lines saying that the strings were from a4. I mean the ones above the strings.

**Comment by khamui** [comment]
@phillimorland

**Comment by phillimorland** [comment]
@khamui just heads up, we don not touch the german wording anymore, that should all be done in Weblate now, not sure if that's reason for fail? Also do you get regex warnings on `make po`, it succeeds so not important but just curious?!

**Comment by khamui** [comment]
> @khamui just heads up, we don not touch the german wording anymore, that should all be done in Weblate now, not sure if that's reason for fail? Also do you get regex warnings on `make po`, it succeeds so not important but just curious?!

@phillimorland okay! Noted with the weblate. I don't get any regex errors.

**Comment by phillimorland** [comment]
to be merged after a4 updates and https://github.com/liqd/a4-meinberlin/pull/4672

**Review by phillimorland — APPROVED** [review]
fixed with new linting

**Pull request merged**